test/Microsoft.DotNet.Cli.Utils.Tests/GivenAProjectToolsCommandResolver.cs

```diff
@@ -282,7 +282,7 @@ public void It_adds_fx_version_as_a_param_when_the_tool_has_the_prefercliruntime
 
             result.Should().NotBeNull();
 
-            result.Args.Should().Contain("--fx-version 1.0.3");
+            result.Args.Should().Contain("--fx-version 1.1.0");
```

Comment:
We probably should change this test. Before, the tool was being built for 1.1 and running on 1.0.3. Now, we should have it be built for 1.0.3 and it will run on 1.1. Basically, what this test does is validate the the CLI tools honor the prefercliruntime file and invoke the tool with the CLI runtime.